### DISCUSSION TITLE
Request that Bats, formula be changed to point to Bats Core

### DISCUSSION THREAD

**Discussion by MichaelDimmitt** [Homebrew/discussions#24] [Everyday usage]
Wanted to reopen the discussion that occured on this ticket:
https://github.com/Homebrew/homebrew-core/issues/27796

Did not know if I should post here in discussions or as a new issue.

The closing action was the statement:
1) project needs significant popularity 
or 
2) project needs to be handed over by the maintainer.

The project has been handed over by the maintainer. ✅  

Here is the ticket where the bats maintainer handed over his project to bats core:
https://github.com/sstephenson/bats/issues/150#issuecomment-186705676
and 
https://github.com/sstephenson/bats/issues/150#issuecomment-323845404

Here is an example where the pre existing bats is causing issues. It includes a deeper discussion.
https://github.com/exercism/bash/pull/445

Hopefully, I can get some direction and thank you for your time.

**Comment by SMillerDev** [comment]
Could you link to the comment where they're specifically mentioning that bats-core is the successor? All I see is a comment stating an intention to transfer ownership and someone creating a new repo.

**Reply by MichaelDimmitt** [reply]
Sure @SMillerDev ,

Here in the opening of the ticket @sstephenson writes:
```
 I’m happy to work closely with you on prioritizing the work and answering any questions you have about the code base.

Beyond that, I’d leave the future direction of the project up to you.

Please use this thread to nominate yourself and tell me why you’d like the job. Let’s get a conversation going.
```

ref - https://github.com/sstephenson/bats/issues/150#issue-134942582

**Reply by SMillerDev** [reply]
An open-ended request for someone to take over isn't quite the same as naming a successor though. While I think it's the spiritual successor to bats I don't think it should be quite named as being the same software.
